src/services/requestErrors.js

```diff
@@ -8,24 +6,28 @@ export default async function createRequestError({
   responseBody = 'N/A',
   responseCode = 'N/A',
 }) {
+  // eslint-disable-next-line global-require
+  const { RequestErrors } = require('../models');
```

Comment:
why is this necessary now?

Reply:
I don't know specifically which change made this required, but there seems to be a recursive include that was making it non-fictional without this change.